jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/perspectives/AbstractPerspective.java

```diff
@@ -107,6 +109,16 @@ protected void setupDocks() {
                                                 getPerspectiveId()).withSize(400).withLabel(Constants.INSTANCE.Filters());
             uberfireDocks.add(basicFiltersDock);
         }
+
+        final String advancedFiltersScreenId = getAdvancedFiltersScreenId();
+        if (advancedFiltersScreenId != null) {
+            advancedFiltersDock = new UberfireDock(UberfireDockPosition.WEST,
+                                                   IconType.SEARCH_PLUS.toString(),
```

Comment:
Just checking the designs, and it seems that they used the _FILTER_ icon with a + sign. Not sure we can replicate this here. Otherwise better double check if this one is ok to use? Not user if using this one would also mean changing the basic filters to use _SEARCH_ instead.